openmetadata-airflow-apis/.coveragerc

```
@@ -0,0 +1,9 @@
+[run]
+source = venv/lib/python3.9/site-packages/openmetadata
```

If we are going to use this configuration in CI this should be:
```suggestion
source = env/lib/python3.9/site-packages/openmetadata
```